templates/acm-certificate-alt-names.yaml

```diff
@@ -0,0 +1,38 @@
+# Setting up a certificate requires a manual step to enter the DNS Validation on godaddy.
```

Comment:
is there an existing folder we can put this file in?  if not them maybe make a folder?

I don't understand the intention of this file.  is it meant to replace the existing acm-certificates.yaml file?  is it meant to provide additional functionality to acm-certifcates.yaml?  is it meant for a completely different use case than acm-certificates.yaml?